lambda/lambda_test.go

```diff
@@ -154,3 +163,112 @@ func TestUnitExecute(t *testing.T) {
 		})
 	})
 }
+
+func TestIntegrationExecute(t *testing.T) {
+
+	mockCtrl := gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	cfg, err := config.GetTestConfig()
+	if err != nil {
+		t.Errorf("Failed to fetch test config: %s", err)
+		t.FailNow()
+	}
+
+	if cfg.NotPopulated() {
+		t.Log("Some config required for integration tests is not populated. Skipping test")
+		t.SkipNow()
```

Comment:
Would it be better to fail, rather than skip the tests?
If the tests are skipped, they are likely to be ignored.

Reply:
If at some point someone decided to add the `test-integration` target to the `test` target, I didn't want to tests to fail should someone not have the required env variables set, so I figured it best to log a message and bail out. Happy to change if you have a massive preference though? 👍